packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -638,6 +640,20 @@ export function TransferPanel() {
       return
     }
 
+    // Make sure Ethereum and an Orbit chain are not selected as a pair.
+    const isEthereumAndOrbitChainPair = [l1Network.id, l2Network.id].every(
```

Comment:
Can't we block it on UI selection in a similar way we used to block selection of One to Nova transfers previously? Then we wouldn't need to handle this case in `transfer` logic, wdyt?